src/main/kotlin/no/nb/bikube/catalogue/collections/repository/CollectionsRepository.kt

```diff
@@ -81,14 +86,15 @@ class CollectionsRepository(
         return getRecordsWebClientRequest(query, db).bodyToMono<CollectionsTermModel>()
     }
 
-    private fun getRecordsWebClientRequest(query: String, db: CollectionsDatabase): WebClient.ResponseSpec {
+    private fun getRecordsWebClientRequest(query: String, db: CollectionsDatabase, fields: String = ""): WebClient.ResponseSpec {
```

Comment:
Just a heads-up, setting fields=empty string means not getting any fields (except @-fields, see response below). Except for getting title, we are only using this function to check if name and publisher exists, which works for fields=empty string. But this might change in the future. Maybe add a comment explaining this?
```
    "adlibJSON": {
        "recordList": [
            {
                "@priref": 4,
                "@created": "2024-03-25T11:05:15",
                "@modification": "2024-03-25T11:05:15",
                "@selected": false
            }
        ],
        "diagnostic": {
        ...
```